src/test/run-make/linker-output-non-utf8/Makefile

```diff
@@ -0,0 +1,12 @@
+-include ../tools.mk
+
+# Make sure we don't ICE if the linker prints a non-UTF-8 error message.
+
+# The zzz it to allow humans to tab complete or glob this thing.
+bad_dir := $(TMPDIR)/zzz$$'\xff'
+
+all:
+	$(RUSTC) library.rs
+	mkdir $(bad_dir)
+	mv $(call STATICLIB,library) $(bad_dir)
```

Comment:
Oh weird! Yeah that documentation should probably be updated!

Could this just hardcode liblibrary.a for now to work on Windows as well? In general we prefer to have tests run everywhere whenever possible.